app/main.py

```diff
@@ -2,4 +2,61 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
+
 # write your code here
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```

Comment:
If argument can be None you must show it type hint. Also, you must specify inner type of list. Fix everywhere.
```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] | None = None) -> None:
```